dnf/cli/commands/autoremove.py

```diff
@@ -37,7 +37,7 @@ class AutoremoveCommand(commands.Command):
                    'autoremove-na': hawkey.FORM_NA,
                    'autoremove-nevra': hawkey.FORM_NEVRA}
 
-    aliases = ('autoremove',) + tuple(nevra_forms.keys())
+    aliases = ('autoremove', 'ar') + tuple(nevra_forms.keys())
```

Comment:
I would not suggest calling this `ar`, and instead use `autorm` if you want a short form for this. Honestly, I'd probably say no short form for this one.